pkg/cmd/docgen/extract/extract.go

```diff
@@ -77,7 +78,8 @@ func GenerateRRNet(bnf []byte) ([]byte, error) {
 	v.Add("options", "factoring")
 	v.Add("options", "inline")
 
-	resp, err := httputil.Post(context.TODO(), rrAddr, "application/x-www-form-urlencoded", strings.NewReader(v.Encode()))
+	httpClient := httputil.NewClientWithTimeout(120 * time.Second)
```

Comment:
if you want something fancy you should make this a flag with `flag.Duration` :P

Reply:
Good call I think it's pretty straight forward to do this as well. Will update.

Reply:
Updated PTAL @otan